### DISCUSSION TITLE
Bun Support

### DISCUSSION THREAD

**Discussion by bnussman** [flash-oss/graphql-upload-minimal#42] [General]
First, just wanted to say think for for maintaining this library! 

I've been working on moving my API to the Bun runtime (https://github.com/bnussman/beep/pull/63) but I have been getting the error "_Request disconnected during file upload stream parsing._" when I attempt file uploads. 

https://github.com/flash-oss/graphql-upload-minimal/blob/d422f5569cd492d47aec0724938d52f2a5c7215c/public/processRequest.js#L346-L348

I didn't want to open an issue because @Jarred-Sumner wants Bun to be responsible for compatibility rather than the package maintainers. 

Has anyone done any investigation into this issue with Bun? I don't necessarily expect `graphql-upload-minimal` to implement a fix, I mostly just wanted to create a discussion to track down the compatibility issues and link related bun issues or PRs.

**Comment by koresar** [comment]
Yeah, it looks like compatibility issue so far.

I am open to PRs. It's not an issue for me to support a few runtimes at once.

**Reply by bnussman** [reply]
I appreciate that! For now, I'll keep an eye on Bun for a fix and I'll update this discussion once a fix lands on Bun's side!
